Integration test test_shutdown fails in container

The integration tests from `tests/integration_tests/functional/test_shut_down.py` fails every time when it is run with the devtool on an i3.metal instance. 
Running the test with `pytest` directly is not triggering the error.
**Command**:
```bash
./tools/devtool -y test 2>&1
```

**Output:**
```bash
=================================== FAILURES ===================================
_________________________ test_reboot[ubuntu_with_ssh] _________________________
integration_tests/functional/test_shut_down.py:51: in test_reboot
    time.sleep(0.01)
E   Failed: Timeout >300.0s
....
....
+++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++

~~~~~~~~~~~~~~~~~~~~~ Stack of Thread-39 (139807611148032) ~~~~~~~~~~~~~~~~~~~~~
  File "/usr/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/dist-packages/paramiko/transport.py", line 1982, in run
    ptype, m = self.packetizer.read_message()
  File "/usr/local/lib/python3.6/dist-packages/paramiko/packet.py", line 441, in read_message
    header = self.read_all(self.__block_size_in, check_rekey=True)
  File "/usr/local/lib/python3.6/dist-packages/paramiko/packet.py", line 288, in read_all
    x = self.__socket.recv(n)

~~~~~~~~~~~~~~~~~~~~~ Stack of Thread-27 (139807585969920) ~~~~~~~~~~~~~~~~~~~~~
  File "/usr/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/dist-packages/paramiko/transport.py", line 1982, in run
    ptype, m = self.packetizer.read_message()
  File "/usr/local/lib/python3.6/dist-packages/paramiko/packet.py", line 441, in read_message
    header = self.read_all(self.__block_size_in, check_rekey=True)
  File "/usr/local/lib/python3.6/dist-packages/paramiko/packet.py", line 288, in read_all
    x = self.__socket.recv(n)
.......
```

--- a/tools/devtool
+++ b/tools/devtool
@@ -382,6 +382,7 @@ run_devctr() {
     # Finally, run the dev container
     #
     docker run "${docker_args[@]}" \
+        --init \
         --rm \
         --volume "$FC_ROOT_DIR:$CTR_FC_ROOT_DIR" \
         --env TEST_MICROVM_IMAGES_S3_BUCKET="$TEST_MICROVM_IMAGES_S3_BUCKET" \

